sale_order_remove_portal_timesheet/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Sale Order Remove Portal Timesheet",
+    "summary": """
+        Sale Order Remove Portal Timesheet
+    """,
+    "author": "Solvos",
+    "license": "LGPL-3",
+    "version": "15.0.1.0.0",
+    "category": "Sales/Sales",
+    "website": "https://github.com/solvosci/slv-sale",
+    "depends": [
+        "sale"
```

Comment:
Wrong inheritance. Since this addon mentions `sale_timesheet` addon, it should depend on it